specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/Global.json

```diff
@@ -0,0 +1,110 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-03-01",
+    "title": "Global API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/billingMeters": {
+      "get": {
+        "tags": [
+          "Global"
+        ],
+        "summary": "Gets a list of meters for a given location.",
+        "description": "Gets a list of meters for a given location.",
+        "operationId": "Global_GetBillingMeters",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
```

Comment:
inline definitions are discouraged. Please create a model definition for this array

Reply:
Also, please add the `x-ms-pageable` extension here.